### PR TITLE
add dns servers to vnet template

### PR DESCRIPTION
**Pull request by timwebster9**
### JIRA link (if applicable) ###

https://dev.azure.com/hmcts/Platform%20Engineering/_workitems/edit/175

### Change description ###

Add DNS servers to VNET ARM template

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```
[AB#175](https://dev.azure.com/hmcts/90472b63-dc8d-4ae6-8819-a32bd211e914/_workitems/edit/175)

### CODE REVIEW

File: templates/arm-vnet-template.json [hunk 1]
```diff
@@ -30,7 +30,12 @@
       "defaultValue": "aks-appgw"
     },
     "appgwSubnetCIDR": {
-      "type": "string"
+      "type": "string",
+      "defaultValue": "10.10.2.0/28"
```
Comment (timja) [inline templates/arm-vnet-template.json]: what is this? we probably shouldn't have a default unless this is to not have a breaking change temporarily

Comment (timwebster9) [reply templates/arm-vnet-template.json]: it was just to make it easier to test - can remove if nobody wants it there?

Comment (timja) [reply templates/arm-vnet-template.json]: hmm, I see, still not really sure we should be letting defaults in =/

Comment (timja) [reply templates/arm-vnet-template.json]: can remove them all separately later, it shouldn't cause any harm since the rest of them have defaults

Comment (timwebster9) [reply templates/arm-vnet-template.json]: yeah we either remove all of them or keep all of them.  It's easy enough to have a parameters file to test with

**Pull request merged**